test/java/seedu/addressbook/FormatterTest.java

```diff
@@ -40,23 +41,30 @@ public void formatPerson() throws Exception {
     public void formatPersons() throws Exception {
         TestDataHelper helper = new TestDataHelper();
         List<Person> persons = helper.generatePersonList(false);
-        String expected = " \t1. Person 1 Phone: 1 Email: 1@email Address: House of 1   Tags: [tag1][tag2]" + NEWLINE
+        String expected = " \t1. Person 1" + NEWLINE + " Phone: 1" + NEWLINE + " Email: 1@email" + NEWLINE
+                + " Address: House of 1" + NEWLINE + " " + NEWLINE + "  Tags: [tag1][tag2]" + NEWLINE
                 + " " + NEWLINE;
         assertEquals(expected, formatter.format(persons));
 
         /**List of 3 people, all fields non-private**/
         persons = helper.generatePersonList(false, false , false);
-        expected = " \t1. Person 1 Phone: 1 Email: 1@email Address: House of 1   Tags: [tag1][tag2]" + NEWLINE
-                + " \t2. Person 2 Phone: 2 Email: 2@email Address: House of 2   Tags: [tag2][tag3]" + NEWLINE
-                + " \t3. Person 3 Phone: 3 Email: 3@email Address: House of 3   Tags: [tag4][tag3]" + NEWLINE
+        expected = " \t1. Person 1" + NEWLINE + " Phone: 1" + NEWLINE + " Email: 1@email" + NEWLINE
+                + " Address: House of 1" + NEWLINE + " " + NEWLINE + "  Tags: [tag1][tag2]" + NEWLINE
+                + " \t2. Person 2" + NEWLINE + " Phone: 2" + NEWLINE + " Email: 2@email" + NEWLINE
+                + " Address: House of 2" + NEWLINE + " " + NEWLINE + "  Tags: [tag2][tag3]" + NEWLINE
+                + " \t3. Person 3" + NEWLINE + " Phone: 3" + NEWLINE + " Email: 3@email" + NEWLINE
+                + " Address: House of 3" + NEWLINE + " " + NEWLINE + "  Tags: [tag4][tag3]" + NEWLINE
```

Comment:
Looks quite fugly. Format Strings will clean it up better

Reply:
Im just ensuring all current tests pass with each update for now.

Reply:
Have not written tests for all my new features

Reply:
Its not planned to be merged by V1.2 right?

Reply:
I can finish up UG,DG but tests i cant complete by v1.2

Reply:
As in, do you want it to be merged by V1.2?